pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/generate/GeneratorPayload.java

```diff
@@ -30,12 +32,22 @@ public GeneratorPayload(NameValue nameValue, Patch patch) {
     private GeneratorPayload(Patch patch) {
         this();
         this.patch = patch;
+        if (patch != null) {
```

Comment:
No change needed in PR, but maybe a TODO is warranted.  The constructor that includes a null value is only used in tests which seems to ignore that a realm and id are both really needed for the update being performed.

Reply:
Ah, I think I can revert my changes in an upcoming PR since I am no longer using GeneratorPayload there.

Reply:
Ah, in an upcoming PR I am no longer using GeneratorPayload so I can revert my changes.